src/components/common/chart/chart.ts

```diff
@@ -0,0 +1,322 @@
+import { Component, Input, OnChanges, OnInit, SimpleChanges } from '@angular/core';
+import ApexCharts from 'apexcharts';
+import { ApexAxisChartSeries, ApexNonAxisChartSeries, ApexOptions, ChartType } from 'ng-apexcharts';
+import { isEqual } from 'lodash-es';
+import { ExaminerRecordData } from '@pages/examiner-records/examiner-records.selector';
+
+@Component({
+  selector: 'chart',
+  templateUrl: 'chart.html',
+})
+export class ChartComponent implements OnInit, OnChanges {
+  @Input() public zoomSize: string = '16px';
```

Comment:
please follow the style for inputs used everywhere else in the app

  @Input()
  examinerId: number;